oidc-exchange.py

```diff
@@ -71,6 +71,9 @@
 * `repository_owner_id`: `{repository_owner_id}`
 * `job_workflow_ref`: `{job_workflow_ref}`
 * `ref`: `{ref}`
+
+See https://docs.pypi.org/trusted-publishers/troubleshooting/ for more help
```

Comment:
Being a sentence, this needs a trailing period:
```suggestion
See https://docs.pypi.org/trusted-publishers/troubleshooting/ for more help.
```